### DISCUSSION TITLE
Buffering PCAPs only if alert hit

### DISCUSSION THREAD

**Discussion by AhmedMontaserT** [Security-Onion-Solutions/securityonion#6648] [Unsupported Versions] [answered]
can i buffer only pcap for alerts ?

**Comment by vijaytejak** [comment]


**Reply by dougburks** [reply]
@vijaytejak Please do not tag individuals unless they have already volunteered to help in the discussion. Thanks.

**Comment by dougburks** [comment, marked as answer]
@AhmedMontaserT Storing pcap only for alerts is problematic for at least a few reasons:
1. Adversaries are going to try to evade your alerts which in this case would evade pcap.
2. In most attack scenarios, the alert (and its underlying pcap) is only going to represent a small percentage of the attacker's activity.

You can tune your pcap storage using BPFs:
https://docs.securityonion.net/en/2.3/bpf.html
